src/main/groovy/org/labkey/gradle/task/StopTomcat.groovy

```diff
@@ -29,7 +29,7 @@ class StopTomcat extends DefaultTask
     @TaskAction
     void action()
     {
-        if (BuildUtils.useEmbeddedTomcat(project))
+        if (BuildUtils.useLocalEmbeddedTomcat(project))
```

Comment:
Someone who has built a standalone distribution that includes the embedded jar and then uses the stopTomcat task will have problems here.  Do we have any documentation or help for this situation?